src/components/Header.js

```diff
@@ -1,6 +1,6 @@
 export default function Header() {
   return (
-    <header className="header-wrapper">
+    <header>
```

Comment:
I  question is `<header>` more semantically correct than simply having the `<h1>` and `<h2>` wrapped in a `<div>`. I will look into for sure.